internal/test/util.go

```diff
@@ -0,0 +1,33 @@
8pNNojrRQJCuhsaggJmXy9Bfqw5BsknRy/hbUf3HSZgK33GyU4ZKj/8CHrhKDH9F7HWonr2DAO/7P4NoPKyIoyP7YIu2/6kULvGjzBSFbfznrIeJlKhx9u0YLsmrnTxVk3wOH34SO/lNSW0iKQdLUdudCzP3sgO2G6pPgjAq8orE+deGTn6o9PiU5Jd9S6tJgpNqEp+nI33gQzltmuhHzuxvMQ68jbGLL/P3248H1u7YqIdHJwwOhz0a+SqWx5dNFDMvI29+V5+T+5sFeIKwPaa8x/6dTc70vYvqs471EsfmLkCip3iCkRX0dSjye2eoeZQpQprvB0eo/IPMqGBw07T4qcW2o0/zFg/maxHsKyay4KWpbxyMj9i7fuHwzboLedPCRIMhoUW47E/ce8w4uZFjhXKtuKSYVOmqi3sIsMa8ViLjAPpO5/9aHidJm6jgNIxq8xxDskLZWSi1r5LwB09si68pPbrtqHK9yFPaKR93z9VhXhuEof/6FXwuZ55w7CZcbyurUIgtZfnK+UJWmbwDYth8uUj+Z2mRyd74npsUFTHuSGBeyuWDdGGCzMg4ZRNxcykqcSH3Ltv+aPooWi0qiiJQuBg7jZ4+loRUtC2Ptur7wr0bhnQb++6/K0J4Pz71QPJ5QzTTUR+Z7++h6g4r7e19tJN8ksPi/SXM5wvr2NWqItBGYpfdBS2vLVVA9U5kb30FiX7Gt35MV5uKbYh9QBirp73fqShopLgGupPvB0h+a+fPaUdkQ9S+aa65ORJ2fWctOIyFLM43vE5Haeyim2OEgkFTG2yMCR8GV84HJ6ta7SqgTXbgvM0oPd4EY71FqtpMQuAUsmFb1VDSm/XQTeJCtDMW0PsHiVVx/cgq4leI3FkOD4LpnMaBTEW4eoCDlyZOR5WuxJ2NvoE3f6tkPomxeH7CkHV+mcRwDOb/s41WIqtZtk6pMFV92FjSit3G4idysjP+qXvQCS3LbAsRz7oKxMOHMeNE/ubdb0EsaoZ7xFAZf7avW4vK6544/o7DGIumIVX2kH76gnnpS7Y7k6QDpGBp8YVCCV/HlmxnaJP0qiGEEGP1zIicYVE0WwtYxyCpgo8gE8CyO++x7KLkf/bFbGLLJZ67UaOsrS88i+6gPyeqFzC6lkXj532lmlwK137vu783Hjqik5LHGDR7t8R5TX7q7JhiOwDHHUjp1/6YpgYsPrE578kSNelCpD3E+yfu2qU9WC/SlRJpvJNpG/stb7SCoEA1i30VGL+U/3tI/CH0A2/nvIt30R3a1fiXS1Vh5NNzx0p/0UkJEbsK6+bw1HhQXTFn48p6QJmg4jQJhUhEQ13hXescGL9jnrMuC07CqsEvdnzlLBZ4nVNC2FSamYpi6V2XPDFU+afLT3Z4VnNtm5Y/9CmM2egMYbHvBGMtHCKXmBn1LiSdvy2LxH8IfcJgWH/aCXuz352F7WVM4OWLQQE2/9Vw2Hr4t5Z3ebcGMpm4f4wiUqq3ew1ra7/6PgxwDuSjtlUsdEehfFWWCx2eiIEJD2aaRpWMgL+xhFBORsia1q6WqQse7XgN+rGnf7OBNL1zS6UO8TarpPvZ0hTF8SPNQtQLR3CsKB8KDPKBWXsqiWayvaUD+/uhqQwV9ShgkiBq9JRW1lwf5FIwgbYvXpf9nmQ/gAfwYeF3FCagoYNAgb8G3w3O8IXHU4zGvqsPflJZzOiHtjHZAM5Yysnq6xkFH9cFIvQtL+Dv4sZT8PEL6dDwlObQawxomQ+om2Uc2eEoQvOdtu9YjOr60Z9NYuUiwtby870ol8YSgmeXzu9g8SYdz1wSVXFLyiTuxlsQ40FGZWpmea/iVdwzh6j51gMGQd7Npug6P8RO9N8Jy50KTzz9i+UUa9otVgrims7sLPzm7KDXHubQzl0XQu8rUfEYu57kRVT7w6Rp8ZPe6LITaQBihNi1gzpkRgSvq7qKBCS7ZuC3NhXgVelXg0V/24f8Z3qbauofxlRCDNTUZjzpyDO/uGniHhd7qwSruuaM9JpnY7JRQguwTkDyh9G1JOaF9ZhPb81fBtYJt8QEMHIQhwYEN8GeJE0Vnm94j9FLCqBRHdMy579bEDWH14mDVJioMZr6sJlp6yiRutnB0l3ee/rgxVW84qJGtRywMMW3c7znoBggNWDfaX+4w00lSXQRoAQr32HtDIm3q0iZkYgqevDQiaTVPv9yWoU/AMrslvm/LYxvXBens/fQU/NQ4ECD7Xb0n7CSKUbV7JdqsWh/L8q1Cuv795Re6omjgnxD+1+8+vCu/DP3kiHlC0kvLDpOOmP5hxyiMsfs6Tlgtd+HZeBRFsSSvGKdgU5FWe6hP0WVgahecTG0MdJ1HHyJ1ggxvCwKS8IlQ1x9QDqmRt7odH3BuLc3v10/Z+4kPsg7XXsbTT1COPl4SrWKqpWBhfqaeEmVoO4WJdjEql7QEfHgkuUc9xjU3n3BxI+Lipkrjzyuz+vv768tlmqu2H4Tb55bzF+byfDINnRgUCHpwucVaxMRanq8LZZnnGfxOuLwGtn6NYjBLIowazG0pjFA77wWh8DCghInj+rPdLPzhRvWxRp/DW8v24D6fcZBI0GQRmOEwM+Gv1n37RlepYg32KjFF1AJJW0wbZRc2BwuEvf0D5ht1rSQ9VjFd2G78oRBeN5F6zkIy7QsRSwESfmS0LB2ZTTdACBtX3n9zX7kwvsNPICySaWOkaxVl+sJMRmhZYPSSPYFFKLly+D2su5oSIctCQG8qO/GQzJWCdQjqn7/WhftYQdmv+kWqQj3zikRCeSMDduXR/nb5IDWx6UQXkCCSZC0Sjl0fVhWzFQMCMGCSqGSIb3DQEJFTEWBBRMnFjR8YjQYmIJvi83EyaCU1yuujApBgkqhkiG9w0BCRQxHB4aAEEAbABpAGEAcwAgAG8AZgAgAGMAZQByAHQwMTAhMAkGBSsOAwIaBQAEFNbPiz55aOKuhh8J9ZFqV84OoBZwBAhu3BRUgoInHQICCAA="
```

Comment:
This cert is a test one I found in a team internal repo. Not a real secret.

Reply:
Is it better to read it from a file (something like config or secret) rather than here?

Reply:
It's unclear from the name alone whether it's a private or public cert.